src/Features/Customer/Dashboard/index.js

```diff
@@ -1,24 +1,92 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
 import { Box, Grid } from '@material-ui/core';
+import { useMutation } from 'react-query';
 import { useDispatch } from 'react-redux';
 
 import MainTab from '../../../components/CardMenus/Tabs';
+import AddEditForm from '../../../components/CommonGridBasedForm';
+import { AUTO_COMPLETE } from '../../../components/CommonGridBasedForm/FieldTypes';
 import TemporaryDrawer from '../../../components/Drawer';
-import { openDrawer } from '../actions';
+import Loader from '../../../components/Loader';
+import NoDataFilter from '../../../components/NoDataFilter';
+import { SelectChangeHandler } from '../../../util/CommonGridBasedFormUtils';
+import { cardData } from '../actions';
```

Comment:
What is this cardData means in actions if you are getting card data it should be like getCardData